### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by andrianarivo**
The things I did in this PR:

1. Linters config.
2. Github flow.
3. Use Webpack
4. Use HTML Webpack Plugin
5. Use Task object (index, description, completed)
6. Load style with Webpack Style/CSS Loader

### CODE REVIEW

File: src/modules/TaskStore.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import Task from './Task.js';
+
+export default class TaskStore {
+  constructor() {
+    this.tasksArray = [new Task(1, 'Task 1'), new Task(2, 'Task 2', true)];
+  }
```
Comment (brytebee) [inline src/modules/TaskStore.js:6]: Great job so far. However, you are required to render the list items in the order of their indexes. 

Consider the code snippet below:

```
 this.tasksArray = [
      new Task(1, "Task 1"),
      new Task(0, "Task 2", true),
      new Task(4, "Task 3", true),
    ].sort((a, b) => a.index - b.index);
```

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @andrianarivo 👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/TaskStore.js (hunk 1 above)
Comment (andrianarivo) [reply src/modules/TaskStore.js:6]: @brytebee your review is subjective. When I render the tasks I sort it. Go look at my render function before I report you to student success.

**Review by cynthiainga — APPROVED** [review]
Hi @andrianarivo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Nice design ✔️ 
- [ ] Correct GitHub flow ✔️ 
- [ ] Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**